doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -99,16 +99,16 @@ julia> UInt
 UInt64
 ```
 
-Larger integer literals that cannot be represented using only 32 bits but can be represented in
-64 bits always create 64-bit integers, regardless of the system type:
+Larger integer literals that cannot be represented using only 32 bits, but can be represented in
+64 bits, always create 64-bit integers, regardless of the system type:
 
 ```jldoctest
 # 32-bit or 64-bit system:
 julia> typeof(3000000000)
 Int64
 ```
 
-Unsigned integers are input and output using the `0x` prefix and hexadecimal (base 16) digits
+Unsigned integers are used as input and output using the `0x` prefix and hexadecimal (base 16) digits
```

Comment:
I was understanding `input` and `output` as verbs, so the initial version looks better to me. Moreover, in this new version, "used" and "using" are used too close in the same sentence.

Reply:
You could say something like "are typed in and printed out using the"...

Reply:
"typed" is pretty overloaded in the programming context. I think the original is quite clear.